gems/aws-sdk-core/spec/aws/cbor/cbor_engine_spec.rb

```diff
@@ -1,4 +1,5 @@
 require_relative '../../spec_helper'
+require 'aws-sdk-core/cbor/cbor_engine'
```

Comment:
Note: These are required because of the way engines are loaded (Ie, we don't just reference the CborEngine, we do a require on the file when trying to determine the default engine.  This means it does not play well with autoloads).

Reply:
Can spec_helper load all the stuff?